processing/topics/roads_bikelanes/bikelanes/BikelaneCategories.lua

```diff
@@ -12,65 +12,77 @@ BikelaneCategory.__index = BikelaneCategory
 -- @param args.condition function
 function BikelaneCategory.new(args)
   local self = setmetatable({}, BikelaneCategory)
+  self.name = args.name
```

Comment:
In processing/topics/roads_bikelanes/bikelanes/BikelaneTodos.lua haben wir `self.key`. Finde ich auch was besser, da `name` für mich Freitext ist. Sollten wir aber gleich halten.

Reply:
wir machen `self.id`, und auch in TODO.